Sources/DistributedActors/ClusterSystem.swift

```diff
@@ -1148,6 +1142,12 @@ extension ClusterSystem {
             )
 
             do {
+                guard let actor = self.resolve(id: recipient) else {
+                    self.log.error("Unable to resolve recipient \(recipient). Message will be dropped: \(invocation)")
+                    self.deadLetters.tell(DeadLetter(invocation, recipient: recipient))
```

Comment:
Should the dead letter happen at sender, after getting back `DeadLetterError`? Or both sender and recipient? 🤔

Reply:
On the recipient I believe, since the recipient will has no function that will throw, so this dead letter informs that system that "someone is calling things on you, but they can't get through to the recipient actor".

And the caller has the function that they called that resulted in a dead letter, that function should _throw_ as it received s the error reply from the dead-letter-ed remoteCall. This way the caller knows it happened, we don't need to log it there.